iree_tests/benchmarks/conftest.py

```diff
@@ -1,8 +1,23 @@
 import pytest
 
 def pytest_addoption(parser):
-    parser.addoption("--goldentime", action="store", default=5565, type=int, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-e2e-ms", action="store", default=1661.2, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-unet-ms", action="store", default=450.4, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-clip-ms", action="store", default=18.9, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-vae-ms", action="store", default=288.5, type=float, help="Golden time to test benchmark")
 
 @pytest.fixture
-def goldentime(request):
-    return request.config.getoption("--goldentime")
+def goldentime_rocm_e2e(request):
+    return request.config.getoption("--goldentime-rocm-e2e-ms")
+
+@pytest.fixture
+def goldentime_rocm_unet(request):
+    return request.config.getoption("--goldentime-rocm-unet-ms")
+
+@pytest.fixture
+def goldentime_rocm_clip(request):
+    return request.config.getoption("--goldentime-rocm-clip-ms")
+
+@pytest.fixture
+def goldentime_rocm_vae(request):
+    return request.config.getoption("--goldentime-rocm-vae-ms")
```

Comment:
At least from what I'm seeing online, the way to pass flags to a pytest program is through the conftest file. I didn't see a (easy) way to have the flags in the test file itself. There are some workarounds (https://stackoverflow.com/questions/59750480/pytest-and-argparse-code-works-in-use-but-tests-failing), but I just went with the pytest documentation for command line flags.

Reply:
Nice, the iree implementation is pretty cool. Thanks for sharing that. We will likely have to do something very similar with regex when we start to benchmark lot more models. I guess for now, with sdxl models and how much we care about every ms, this will do

Reply:
https://docs.python.org/3/library/os.html#os.environ environment variables would work, yeah?

Reply:
Yup environment variables would work. I feel like we already have a good amount of them in the yml file. As a user, I prefer command line flags lol (also just want people to just have to edit that one line in the yml file whenever the golden values are changed with perf boosts). But, maybe that's just me. Fine with either